<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginClass</class>
 <widget class="QDialog" name="LoginClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginClass</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>密 码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoRepeatInterval">
    <number>1000</number>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoRepeatInterval">
    <number>1000</number>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitBtn</sender>
   <signal>clicked()</signal>
   <receiver>LoginClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
